content/docs/command-reference/exp/run.md

```diff
@@ -73,16 +75,22 @@ committing them to the Git repo. Unnecessary ones can be [cleared] with
 
 - `-S [<filename>:]<override_pattern>`,
   `--set-param [<filename>:]<override_pattern>` - set the value of existing
-  `dvc params` for this experiment. `filename` can be any valid params file
-  (`params.yaml` by default). This will override the param file before running
-  the experiment.
+  `dvc params` for this experiment.
 
-  For example, to override the value an existing param, you can use
-  `--set-param <param_name>=<param_value>`.
+  This will update the param file **before** running the experiment.
 
-  Valid `<override_pattern>` values are defined in the
-  [Hydra Override Grammar](https://hydra.cc/docs/advanced/override_grammar/basic/#modifying-the-config-object).
-  Patterns modifying Hydra's defaults list are not supported.
+  For example, to modify the value of an existing param, you can use
+  `-S <name>=<value>`.
+
+  You can optionally provide a prefix `[<filename>:]` in order to edit a
+  specific `dvc params` file. If not provided, `params.yaml` will be used as
+  default.
+
+  Valid `<override_pattern>` values are defined in the [Hydra Override Grammar].
+
+  Using
+  [Hydra's Sweep overrides](https://hydra.cc/docs/advanced/override_grammar/extended/#sweeps)
+  requires the `--queue` option to be also provided.
```

Comment:
Do you think we need to mention here which sweep overrides (choices and range) are supported?

Reply:
Yes. I will rephrase